lib/beaker/test_suite.rb

```diff
@@ -310,6 +311,7 @@ def run
       # has not studied the implementation in detail. --daniel 2011-03-14
       @test_suite_results.summarize( Logger.new(log_path("#{name}-summary.txt", @options[:log_dated_dir]), STDOUT) )
       @test_suite_results.write_junit_xml( log_path(@options[:xml_file], @options[:xml_dated_dir]) )
+      @test_suite_results.write_junit_xml( log_path(@options[:xml_time], @options[:xml_dated_dir]), True ) if @options[:xml_time_enabled]
```

Comment:
Because @anodelman let me know that there is interest in preserving runtime order, so we can't just do one or the other, we have to have a way to do both.